modules/backend/hooks/useAxiosResponseInterceptors.ts

```diff
@@ -8,55 +8,53 @@ import { axiosInstance } from '@/modules/backend/axios-instance'
 
 // https://dev.to/arianhamdi/react-hooks-in-axios-interceptors-3e1h
 
+const successInterceptor = (response: AxiosResponse) => {
+  return response
+}
```

Comment:
I would keep both interceptors together inside `useAxiosResponseInterceptors` function and disable the eslint rule that forces us to put it outside locally.